frontend/src/metabase/nav/containers/MainNavbar/SidebarItems/SidebarCollectionLink.tsx

```diff
@@ -21,23 +21,49 @@ import {
   SidebarIcon,
 } from "./SidebarItems.styled";
 
-// eslint-disable-next-line react/display-name
-const SidebarCollectionLink = React.forwardRef<HTMLLIElement, TreeNodeProps>(
+type DroppableProps = {
+  hovered: boolean;
+  highlighted: boolean;
+};
+
+type Props = DroppableProps &
+  Omit<TreeNodeProps, "item"> & {
+    collection: Collection;
+  };
+
+const TIME_BEFORE_EXPANDING_ON_HOVER = 500;
```

Comment:
```suggestion
const TIME_BEFORE_EXPANDING_ON_HOVER = 600;
```

Just a slight increase seems to feel a bit nicer.